src/planwise/client/scenarios/handlers.cljs

```diff
@@ -325,3 +326,44 @@
    (assoc db
           :view-state       :changeset-dialog
           :changeset-dialog change)))
+
+(rf/reg-event-db
+ :scenarios/edit-change
+ in-scenarios
+ (fn [db [_ change]]
+   (assoc db
+          :view-state       :changeset-dialog
+          :changeset-dialog change)))
+
+(rf/reg-event-fx
+ :scenarios/delete-current-scenario
+ in-scenarios
+ (fn [{:keys [db]} [_]]
+   (let [id (get-in db [:current-scenario :id])]
+     {:db  (assoc db :view-state :delete-scenario)
+      :api (assoc (api/delete-scenario id)
+                  :on-success [:scenarios/load-scenarios])})))
+
+(rf/reg-event-db
+ :scenarios/open-delete-dialog
+ in-scenarios
+ (fn [db [_]]
+   (assoc db
+          :view-state :delete-scenario)))
+
+(rf/reg-event-db
+ :scenarios/open-delete-dialog
+ in-scenarios
+ (fn [db [_]]
+   (assoc db
+          :view-state :delete-scenario)))
+
+;ASK state for showing settings
```

Comment:
Is is the expected state when showing one menu (scenario settings) or the other (creating-provider/line 261)?